solr/core/src/java/org/apache/solr/packagemanager/PackageManager.java

```diff
@@ -121,25 +121,44 @@ public void close() throws IOException {
     Map<String, SolrPackageInstance> ret = new HashMap<String, SolrPackageInstance>();
```

Comment:
nitpick: `Map<String, SolrPackageInstance> ret = new HashMap<>();`

Also, lines below could be:
```
packages.forEach( (k,v) -> {
     if (Strings.isNullOrEmpty(packageName) == false && packages.get(packageName) != null) {
         ret.put(packageName, getPackageInstance(packageName, packages.get(packageName)));
     }
});

Reply:
Couldn't use forEach since the packages variable needed inside the scope wasn't final.